edbdeploy/options.py

```diff
@@ -48,7 +48,7 @@ class ReferenceArchitectureOptionDBaaS:
 
 
 class OSOption:
-    choices = ['CentOS7', 'CentOS8', 'RedHat7', 'RedHat8', 'RockyLinux']
+    choices = ['CentOS7', 'CentOS8', 'RedHat7', 'RedHat8', 'RockyLinux8']
     default = 'CentOS8'
```

Comment:
What about making RockyLinux8 the new default value?

Reply:
Fixed